specification/_doc_ids/table.csv

```diff
@@ -405,6 +405,7 @@ query-dsl-wildcard-query,https://www.elastic.co/guide/en/elasticsearch/reference
 query-dsl-wrapper-query,https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/query-dsl-wrapper-query.html
 query-dsl,https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/query-dsl.html
 realtime,https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/docs-get.html#realtime
+redact-processor,https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/redact-processor.html
```

Comment:
Unrelated, but I think I forgot to add these urls in my previous PR where I added the HtmlStripProcessor 😅